examples/lm1b/input_pipeline.py

```diff
@@ -51,32 +51,25 @@ def train_and_eval_dataset(dataset_name,
          ie., a pair of lists with input and target feature names.
   """
   dataset_builder = tfds.builder(dataset_name, data_dir=data_dir)
+  dataset_builder.download_and_prepare(download_dir=data_dir)
   info = dataset_builder.info
   splits = dataset_builder.info.splits
   if tfds.Split.TRAIN not in splits:
     raise ValueError("To train we require a train split in the dataset.")
-  train_split = tfds.Split.TRAIN
+  train_split = 'train'
   if eval_holdout_size > 0:
     holdout_percentage = int(eval_holdout_size * 100.0)
     train_percentage = 100 - holdout_percentage
-    train_split = tfds.Split.TRAIN.subsplit(tfds.percent[:train_percentage])
-    eval_split = tfds.Split.TRAIN.subsplit(tfds.percent[train_percentage:])
+    train_split = 'train[:{}]'.format(train_percentage)
```

Comment:
I believe that this is not the correct split definition. It will take `train_percentage` examples instead of `train_percentage` percentage of the entire split. Please use the specification in the links I provided earlier.

Can you also provide TensorBoard logs confirming that the metrics after this change are in line with what `README.md` reports?

Reply:
```
train_split = tfds.percent[:train_percentage]
train = dataset_builder.as_dataset(split=train_split, shuffle_files=train_shuffle_files)
``` 
like that?

Reply:
Sorry, should have been more clear. I meant that you're missing the `%`character. So
```
train_split = 'train[:{}%]'.format(train_percentage)
```

Reply:
Ohh I missed it. Thank you! 
 I reseted the wrong commit and forced to push. I think it's solved.

Reply:
@AlexeyG I haven't got a powerful computer so I can't work with a machine like in the report. Also a small computer for training `cifar10`.